### PR TITLE
Bucash1.3.0.0 relnotes

### PR DESCRIPTION
**Pull request by sickpig**

### CODE REVIEW

File: doc/release-notes/release-notes-bucash1.3.0.0.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+
+If you are upgrading from a release older than 1.1.2.0, your UTXO database will be converted
+to a new format. This step could take a variable amount of time that will depend
+on the performance of the hardware you are using.
+
+Downgrade
+---------
+
+In case you decide to downgrade from BUcash 1.2.0.1 to a version older than 1.1.2.0
+will need to run the old release using `-reindex` option so that the
```
Comment (ptschip) [inline doc/release-notes/release-notes-bucash1.3.0.0.md]: nit:  this should probably be  "from BUcash 1.2.0.1, or greater, to a version..."

**Review by ptschip — APPROVED** [review]
other than the nit this looks good.

File: doc/release-notes/release-notes-bucash1.3.0.0.md (hunk 1 above)
Comment (sickpig) [reply doc/release-notes/release-notes-bucash1.3.0.0.md]: agreed

Comment (sickpig) [reply doc/release-notes/release-notes-bucash1.3.0.0.md]: change pushed

**Review by ptschip — APPROVED** [review]
re-approve

**Pull request merged**